.github/workflows/cache-images.yaml

```diff
@@ -0,0 +1,49 @@
+name: Cache Anbox Cloud images
+
+on:
+  pull_request:
+  workflow_dispatch:
```

Comment:
If we run this on a daily basis any manual invocation will be quickly overwritten unless we add the Android version to the cache artifact.